source/query/atlas-search.txt

```diff
@@ -25,32 +39,28 @@ Steps
    :style: normal
 
    .. step:: Find the index you want to use in your query
-      
-      a. From the :guilabel:`Indexes` tab, click :guilabel:`Search Indexes`.
-      #. Hover over the index you want to use in your query.
 
-   .. step:: Run the query
+      From the :guilabel:`Indexes` tab, click :guilabel:`Search Indexes`. 
+      Hover over the search index you want to use in your query. 
       
-      a. Click the :guilabel:`Aggregate` button.
-
-         |compass| switches to the :guilabel:`Aggregations` tab where you 
-         can run your query. 
-        
-      #. Construct your query.
+   .. step:: Open the aggregation pipeline builder
+      
+      Click the :guilabel:`Aggregate` button that appears when you hover over 
+      your search index.
          
-         By default, |compass| selects the :pipeline:`$search` stage for
-         your query. You can select the  :pipeline:`$searchMeta` stage
-         to use in your query. If you are using a ``vectorSearch`` type
-         index in your query, you can only use the
-         :pipeline:`$vectorSearch` stage in your query.  
+   .. step:: Construct your query
+      
+      By default, |compass| selects the :pipeline:`$search` stage for your 
+      query. You can select the  :pipeline:`$searchMeta` stage to use in your 
+      query. If you are using a ``vectorSearch`` type index in your query, you 
+      can only use the :pipeline:`$vectorSearch` stage in your query. 
```

Comment:
I know this was in the docs previously, but this actually isn't true. You can include other stages in the agg pipeline alongside $vectorSearch. Would it be convenient to make this change while we're here, or should I file a followup ticket?

Reply:
Will make this change here!